openmdao/visualization/n2_viewer/turbojet.py

```diff
@@ -0,0 +1,30 @@
+import openmdao.api as om
```

Comment:
Filename is incorrect as described before.  And why are we duplicating this file in 2 places?